screening/modular-open-source-identity-plaform.json

```diff
@@ -0,0 +1,130 @@
+{
+  "name": "Modular Open Source Identity Plaform",
+  "ownership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.mosip.io/resource/ip-policy-trademark-and-copyright"
+  },
+  "dependencies": {
+    "mandatoryDepsCreateMoreRestrictions": "Yes",
+    "isSoftwarePltIndependent": "Yes",
+    "pltIndependenceDesc": "For evey closed source software and hardware component that MOSIP interacts with, we provide standard interfaces for interaction. Either by using existing open standards (ex: For HSM we use JCE), or by defining standard interfaces for conformance (ex: MOSIP biometrics interfaces)"
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "docs.mosip.io"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "nonPIIAccessMechanism": "The project allows each deployment to define what data they collect. Some of these could be non-PII data. "
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Unknown",
```

Comment:
An `Unknown` answer is not valid here. Or rather, if they don't know whether they are privacy compliant or not, then we cannot complete our screening as a DPG

Reply:
Let me write to them to clarify their status on privacy compliance. Shall revert ASAP.

Reply:
MOSIP has now provided official information on privacy compliance and can be found in this commit 0499368.